src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,25 @@
+import { FilterTypes } from '../types/FIlterTypes';
+import { Todo } from '../types/Todo';
+
+export const getFilteredTodos = (todos: Todo[], filterType: FilterTypes) => {
+  let filteredTodos = todos;
+
+  switch (filterType) {
+    case FilterTypes.ACTIVE:
+      filteredTodos = filteredTodos.filter(todo => !todo.completed);
+      break;
+
+    case FilterTypes.COMPLETED:
+      filteredTodos = filteredTodos.filter(todo => todo.completed);
+      break;
+
+    case FilterTypes.ALL:
+      filteredTodos = todos;
+      break;
+
+    default:
```

Comment:
you will never have unexpected type, thanks for typescript, so I think we can move `All` to default case